clean-modular-code/python-dry-modular-code.md

```diff
@@ -132,17 +132,17 @@ to both understand and use. If you need to change the calculation itself, you ca
 
 ```python
 temp = 55
-new_temp = fahr_to_kelvin(fahr = temp)
+new_temp = convert_fahrenheit_to_kelvin(temp)
```

Comment:
I like how expressive these names are. My only concern is they are so long! for the lessons we could do this but in my code I find really long names to make it harder to read too. @willingc do you have any thoughts on this? i love the use of a verb for the function `convert_fahr_to_kelv` might be a bit shorter